src/profiling/intro.md

```diff
@@ -5,6 +5,39 @@ By examining where Rspack spends its time, we can gain insights into how to impr
 
 <!-- toc -->
 
+## Application-level Tracing
```

Comment:
Tracing in Rspack is rough. We might need a refactor for it.

Reply:
In [software engineering](https://en.wikipedia.org/wiki/Software_engineering), tracing involves a specialized use of [logging](https://en.wikipedia.org/wiki/Computer_data_logging) to record information about a program's execution.

Tracing is enough. We should avoid writing Rspack everywhere.

Reply:
I'm not specifying `tracing` the crate. I'm talking about the way Rspack uses and organizes, which doesn't help to debug.